eclair-core/src/test/resources/logback-test.xml

```diff
@@ -50,9 +50,9 @@
     <logger name="fr.acinq.eclair.db.FileBackupHandler" level="OFF"/>
 
     <root level="INFO">
-        <!--appender-ref ref="FILE"/>
-        <appender-ref ref="CONSOLEWARN"/>
-        <appender-ref ref="CONSOLE"/-->
+        <!--<appender-ref ref="FILE"/>-->
+        <!--<appender-ref ref="CONSOLEWARN"/>-->
+        <!--<appender-ref ref="CONSOLE"/>-->
```

Comment:
Is this to make intellij auto-uncomment work?

Reply:
Exactly, I was annoyed at having to "manually" fix those every time I wanted to add logs to tests :)
Now it works with IntelliJ (and other editors such as VS code) shortcuts.